app/dao/notifications_dao.py

```diff
@@ -408,3 +413,34 @@ def get_april_fools(year):
     """
     return pytz.timezone('Europe/London').localize(datetime(year, 4, 1, 0, 0, 0)).astimezone(pytz.UTC).replace(
         tzinfo=None)
+
+
+def get_total_sent_notifications_in_date_range(start_date, end_date, notification_type):
+    result = db.session.query(
+        func.count(NotificationHistory.id).label('count')
+    ).filter(
+        NotificationHistory.key_type != KEY_TYPE_TEST,
+        NotificationHistory.status.in_(NOTIFICATION_STATUS_TYPES_COMPLETED),
+        NotificationHistory.created_at >= start_date,
+        NotificationHistory.created_at <= end_date,
```

Comment:
this should refer `updated_at` - or a notification that is created at 11:59 on 1st jan but not delivered until 12:31 2nd jan (ie: after the job collecting for 1st jan has finished running) will never get captured